rest/rest/src/main/java/io/syndesis/rest/v1/handler/integration/IntegrationHandler.java

```diff
@@ -197,6 +197,7 @@ public Integration create(@Context SecurityContext sec, @ConvertGroup(from = Def
             .createdDate(rightNow)
             .currentStatus(determineCurrentStatus(encryptedIntegration))
             .userId(sec.getUserPrincipal().getName())
+            .stepsDone(new ArrayList<>())
```

Comment:
Shouldn't this be the default in the constructor of an integration ?

Reply:
Not the case when we are creating an integration from an import.

Reply:
Not the case when we are creating an integration from an import.